gsoc/gsoc2016/ideas_astropy.md

```diff
@@ -150,3 +150,31 @@ This project includes looking at other astronomy codes to see how they tackle th
 
 See https://github.com/OpenAstronomy/openastronomy.github.io/pull/27 for a discussion of some problems and possible solutions that will be addressed in this project.
 
+
+### Bridge sherpa and astropy fitting
+
+*Suggested Mentor(s):* D. Bourke, T. Aldcroft, H. M. Guenther
+
+*Difficulty:* Expert or better
+
+*Astronomy knowledge needed:* fitting functions and statistics
+
+*Programming skills:* Python, C, Cython
+
+#### Description
+
+
+Both astropy and Sherpa (https://github.com/sherpa/sherpa/) provide modelling and fitting capabilities; however, Sherpa's features are way
+more advanced. Sherpa provides far more build-in models, a larger choice of optimizers and a real variety
+of fit statistics.
+Unfortunately, Sherpa is less well known and (for partly historical reasons), the interface to do
```

Comment:
True (and I do that in my code), but there are probably very few non-CXC users who even know about it.